api/types/swarm/task.go

```diff
@@ -98,6 +98,13 @@ type Resources struct {
 	GenericResources []GenericResource `json:",omitempty"`
 }
 
+// ResourceLimit describes limits on resources which can be requested by a task.
+type ResourceLimit struct {
+	NanoCPUs    int64 `json:",omitempty"`
+	MemoryBytes int64 `json:",omitempty"`
+	PidsLimit   int64 `json:",omitempty"`
```

Comment:
Should we name this `Pids` or `PidsLimit` ? Wasn't sure (and naming is hard)

@cpuguy83 @tonistiigi

Reply:
Pids

Reply:
updated 👍